cpp/src/interop/to_arrow_device.cu

```diff
@@ -412,9 +228,12 @@ int dispatch_to_arrow_device::operator()<numeric::decimal128>(cudf::column&& col
                                                               rmm::mr::device_memory_resource* mr,
                                                               ArrowArray* out)
 {
-  using DeviceType = __int128_t;
-  return decimals_to_arrow<DeviceType>(
-    std::move(column), cudf::detail::max_precision<DeviceType>(), stream, mr, out);
+  nanoarrow::UniqueArray tmp;
```

Comment:
Can this overload now be removed and just use the default version of the function? I think the code paths are now identical for decimal128 and all the rep compatible types.

Reply:
Oh wow. I did not even notice that. I will look into it.